docs/ENV.md

```diff
@@ -54,6 +54,19 @@ These variables are specifically for a single language or tool.
+| COCOA_INCLUDED_TARGETS                  | Comma-separated list of target to include for resolution of dependencies. Usage is the same as `COCOA_EXCLUDED_TARGETS` above.                                                                                                                                                                                                                                                                                                                                                                                                       |
+| COCOA_MERGE_SUBSPECS                    | Should all sub-specs (and their dependencies) be merged into the root spec. This can be useful if you are importing your SBOM into a tool that doesn't handle sub-specs as separate dependencies (eg dependency-track). Defaults to `false`.                                                                                                                                                                                                                                                                                         |
+| COCOA_PODSPEC_JSON_REPLACEMENTS         | Comma-separated list of 'text_to_find=replacement'. This can be used if you have local pods in JSON format that are not (correctly) parsable by the `pod ipc sepc`-command, eg because some variables or functions would be inserted by the `Podfile` -- users of react-native / expo might need this. This will be interpreted as a `regex` if the 'text_to_find' is both prefixed and suffixed with a `/` and then references like `$1` can be used in the 'replacement'. To find or insert a newline, use the String `<NEWLINE>`. |
+| COCOA_PODSPEC_REPLACEMENTS              | Similar to `COCOA_PODSPEC_JSON_REPLACEMENTS`, but for local pods that are in the 'podspec' format.                                                                                                                                                                                                                                                                                                                                                                                                                                   |
+| COCOA_RESOLVE_FROM_NODE                 | If some of your local pods are included from node (eg when using expo or react-native), set this to true to use the npm-packages as your dependencies. The big advantage of this, is that the generated purls will be of actually known components (eg in OSS Index) instead of generic names for the packages.                                                                                                                                                                                                                      |
```

Comment:
Can we default this to true?

Reply:
Are you sure? It might confuse users that they have mixed packages... Then again, users that are building with expo should probably know this is happening.
Also, if we change it here, we should probably change it for gradle as well, don't you think?

Reply:
Good idea. Let's change both places to true